config.go

```diff
@@ -43,7 +43,7 @@ const (
 	defaultBlockMaxSize      = 375000
 	blockMaxSizeMin          = 1000
 	blockMaxSizeMax          = wire.MaxBlockPayload - 1000
-	defaultBlockPrioritySize = 50000
+	defaultBlockPrioritySize = 20000
```

Comment:
Why this value? Will priority space still be used contrary to the first PR comment?